### PR TITLE
minor changes, single message format

### PR DESCRIPTION
**Pull request by kkohbrok**

### CODE REVIEW

File: draft-robert-mimi-delivery-service.md [hunk 1]
```diff
@@ -123,6 +127,9 @@ Client           Delivery Service      Queueing Service
 ~~~
 {: title="Client/Delivery Service communication with fanout" }
 
+In the federated case, messages to a "foreign" Delivery Service can be proxied
```
Comment (raphaelrobert) [inline draft-robert-mimi-delivery-service.md:130]: We should re-use existing terminology and either call this "guest" or "participating" DS.

File: draft-robert-mimi-delivery-service.md [hunk 2]
```diff
@@ -62,6 +62,10 @@ several messaging services. In particular, it offers the following features:
  * Network fluid: While the Delivery Service would typically run as a
    server-side component, the only requirement is that it is accesible by all
    clients.
+ * Transport agnostic: Messages between clients and the Delivery Service can be
+   sent via an arbitrary transport protocol. Additionally, in the federated
+   case, client messages to a foreign server can be forwarded by a client's
```
Comment (raphaelrobert) [inline draft-robert-mimi-delivery-service.md:67]: See comment below regarding "foreign"

File: draft-robert-mimi-delivery-service.md [hunk 3]
```diff
@@ -148,9 +155,18 @@ Client           Delivery Service      Queueing Service
...
+authentication and client-level authentication.
+
+Client-level authentication is used for client-specific group operations. Here,
+clients authenticate to the Delivery Service using the signature key from their
 respective leaf node in the group and a client-specific identifier.
 
+User-level authentication allows clients, for example, to prove that they belong
+to a certain user they add themselves to a group using server-assistance. To
+achieve user-level authentication, clients authenticate using a user-specific
+public key and use the hash of that key as identifier.
```
Comment (raphaelrobert) [inline draft-robert-mimi-delivery-service.md:168]: I guess this is more the for the sake of completeness, we don't actually need user keys for anything at the moment.

**Review by raphaelrobert — CHANGES_REQUESTED** [review]
Thanks, just some nits

File: draft-robert-mimi-delivery-service.md (hunk 3 above)
Comment (kkohbrok) [reply draft-robert-mimi-delivery-service.md:168]: Fair enough. Let's take one thing at a time.

**Review by raphaelrobert — APPROVED** [review]

**Pull request merged**